en/rocky/8/guides/rocky-8-installation.md

```diff
@@ -68,159 +67,190 @@ Insert and boot off the installation medium (optical disk, USB flash drive, and
 
 Once booted, you'll be presented with a welcome splash screen.
 
-![Rocky Linux installation splash screen](images/installation-F01.png)
+![Welcome screen](images/installation-F01.png)
 
-If you do not press any key, the prompt will begin a countdown, after which the installation process will start by booting the highlighted Test This Media & Install Rocky Linux 8 ... option. You can also press enter to start the process immediately.
+If you do not press any key, the prompt will begin a countdown, after which the installation process will start by booting the highlighted `Test this media & install Rocky Linux 8` ... option. You can also press <kbd>enter</kbd> to start the process immediately.
 
```

Comment:
Yes it should looks like something like that : 
![image](https://user-images.githubusercontent.com/3603739/119095981-46ce4400-ba13-11eb-8c8d-776ae63bc349.png)

Reply:
The rendering in github is like that:
![image](https://user-images.githubusercontent.com/3603739/119096242-a167a000-ba13-11eb-98df-0d65cf6a0657.png)
I like it because it gives us the opportunity to higligh button actions with kdb and UI Parts with italic.
This way people understand they have to click and can recognize UI elements.

Reply:
Have a look here : 
https://github.com/rocky-linux/documentation/blob/11771acb6130782442c1c48551791850f8ff3712/en/rocky/8/guides/rocky-8-installation.md